packetbeat/protos/http/_meta/fields.yml

```diff
@@ -46,7 +46,3 @@
                 Which headers to capture is configurable. If headers with the
                 same header name are present in the message, they will be separated
                 by commas.
-            - name: body
```

Comment:
@adriansr Ok, this now becomse interesting. We had it as text in packetbeat :-( So if we unify it which we should do, we need to do a breaking change for at least one of them :-(